homeassistant/components/sensor/google_travel_time.py

```diff
@@ -186,6 +216,53 @@ def update(self):
         elif atime is not None:
             options_copy['arrival_time'] = atime
 
-        self._matrix = self._client.distance_matrix(self._origin,
-                                                    self._destination,
-                                                    **options_copy)
+        # Convert device_trackers to google friendly location
+        if hasattr(self, '_origin_entity_id'):
+            self._origin = self._get_location_from_entity(
+                self._origin_entity_id
+            )
+
+        if hasattr(self, '_destination_entity_id'):
+            self._destination = self._get_location_from_entity(
+                self._destination_entity_id
+            )
+
+        if self._destination is not None and self._origin is not None:
+            self._matrix = self._client.distance_matrix(self._origin,
```

Comment:
I have made this change as it may actually be needed for states that are already set by the zone they are in.

The documentation update will mention this to let people know that friendly names should not be duplicated for zones if using this feature.